client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -471,12 +471,11 @@ export const SeoForm = React.createClass( {
 						status="is-warning"
 						showDismiss={ false }
 						text={ translate(
-							'SEO settings are disabled because the ' +
-							'site visibility is not set to Public.'
+							"SEO settings aren't recognized by search engines while your site is Private."
 						) }
 					>
 						<NoticeAction href={ generalTabUrl }>
-							{ translate( 'View Settings' ) }
+							{ translate( 'View privacy settings' ) }
```

Comment:
@michelleweber what do you think about `Privacy Settings` instead of `View privacy settings`, since we already have translations for that in place?